akka-remote/src/main/resources/reference.conf

```diff
@@ -93,6 +93,28 @@ akka {
       "akka.actor.ActorKilledException" = akka-misc
       "akka.actor.InvalidActorNameException" = akka-misc
       "akka.actor.InvalidMessageException" = akka-misc
+
+      "akka.actor.LocalScope$" = akka-misc
+      "akka.remote.RemoteScope" = akka-misc
+
+      "com.typesafe.config.impl.SimpleConfig" = akka-misc
```

Comment:
is this needed? is it ambiguous with `Serializable` otherwise?

`SimpleConfig implements Config`

Are there other implementations of `Config` that would have similar problem?